plugins/powerd/fu-plugin-powerd.c

```diff
@@ -74,6 +74,21 @@ fu_plugin_startup (FuPlugin *plugin, GError **error)
 	return TRUE;
 }
 
+static gboolean
+fu_plugin_inhibit_suspend (GError **error)
+{
+	g_autofree gchar *lockdir = NULL;
+	g_autofree gchar *inhibitsuspend_filename = NULL;
+        g_autofree gchar *getpid_str = NULL;
+
+	lockdir = fu_common_get_path (FU_PATH_KIND_LOCKDIR);
+	inhibitsuspend_filename =
+	    g_build_filename (lockdir, "power_override", "fwupd.lock", NULL);
+        getpid_str = g_strdup_printf ("%d", getpid ());
+	return g_file_set_contents (
```

Comment:
Same styling comment as above regarding indenting to line up arguments with the (